silkworm/silkrpc/commands/ots_api.cpp

```diff
@@ -844,14 +844,14 @@ Task<bool> OtsRpcApi::trace_blocks(
     co_return has_more;
 }
 
-Task<void> OtsRpcApi::search_trace_block(ethdb::Transaction& tx, evmc::address address, unsigned long index, uint64_t block_number, std::vector<TransactionsWithReceipts>& results) {
+Task<void> OtsRpcApi::search_trace_block(ethdb::Transaction& tx, evmc::address address, unsigned long index, BlockNum block_number, std::vector<TransactionsWithReceipts>& results) {
     TransactionsWithReceipts transactions_with_receipts;
     co_await trace_block(tx, block_number, address, transactions_with_receipts);
     results[index] = transactions_with_receipts;
     co_return;
 }
 
-Task<void> OtsRpcApi::trace_block(ethdb::Transaction& tx, uint64_t block_number, evmc::address search_addr, TransactionsWithReceipts& results) {
+Task<void> OtsRpcApi::trace_block(ethdb::Transaction& tx, BlockNum block_number, evmc::address search_addr, TransactionsWithReceipts& results) {
```

Comment:
The same change should be applied also in `ots_api.hpp`

Reply:
OK